ntbs-integration-tests/LabResultsPage/LabResultsPageTests.cs

```diff
@@ -37,6 +36,9 @@ public async Task NhsUser_CanViewSpecimensAccordingToPermissions()
                 // Assert
                 Assert.Equal(HttpStatusCode.OK, response.StatusCode);
                 
+                // var specimenDetailsSections = document.QuerySelectorAll(".specimen-details");
+                // Assert.Equal(expectedLabReferenceNumbers.Count, specimenDetailsSections.Length);
```

Comment:
Is this going to be used in the next tickets?

Reply:
Looks like in the PHE user test it's not commented out - was this by accident then?